「サーバを作ろう！」のページを追加

takarada opened the pull request:
「サーバを作ろう！」のページを追加しました。

当該ページには
・AWSコンソールを開いてEC2のインスタンスを立てる手順
・作ったインスタンスにSSH接続させる手順
・インスタンスにnginxをインストールする手順
を記載しています。

alf-dev committed:
「サーバを作ろう！」のページを追加

y-tac-d commented on handson/server/aws_ec2/server_1.md:
```diff
@@ -0,0 +1,255 @@
+公式の手順が一番わかりやすいので、そちらを参照
+<a href="https://aws.amazon.com/jp/register-flow/" target="_blank">AWS アカウント作成の流れ</a>
+
+
```
@takarada  リージョンを選ぶ手順を追加して、"ハンズオンで利用するリージョン”も指定してあげてください。東京でいいとはおもう

y-tac-d commented on handson/server/aws_ec2/server_1.md:
```diff
@@ -0,0 +1,255 @@
+
+
+
+この時、ソースのところに「0.0.0.0/0」と表記されています。
```
自身の接続IPの確認手順と修正例を記述してください（多分ソースIPの取得方法がわかんない人いるとおもう）

y-tac-d commented on handson/server/aws_ec2/server_1.md:
````diff
@@ -0,0 +1,255 @@
+```
+
+これで設定は終わりです。
+ブラウザからhttpで作成したインスタンスにアクセスすれば、以下画面が表示されます。
````
ブラウザからHTTPアクセスするやり方を記述してください

y-tac-d replied:
修正確認しました、がEC2の自動ドメイン経由で確認する方法も追記をお願いします！

y-tac-d reviewed (changes requested):
確認しました！修正をお願いします！

alf-dev committed:
・リージョンを選ぶ手順を追加
・もともと文末に書いていたリージョンに関する補足を削除（今回追加した手順とかぶるので）
・自身の接続IPの確認方法と修正方法を記載
・ブラウザからアクセスしやすいようにURLを記載

alf-dev committed:
・インスタンスにアクセスする手法をSSHクライアントから、AWS Session Managerを使った方法に変更
・ブラウザでアクセスする手法にパブリックIPv4アドレスを使った方法を追記
・マークダウンのリンクの書き方を統一
・その他文言の微修正

alf-dev committed:
文言修正

takarada commented:
@y-tac-d 
確認ありがとうございます。
以下修正を施しました、ご確認くださいませー

    ・インスタンスにアクセスする手法をSSHクライアントから、AWS Session Managerを使った方法に変更
    ・ブラウザでアクセスする手法にEC2の自動ドメイン経由で確認する方法を追記
    ・マークダウンのリンクの書き方を統一
    ・その他文言の微修正

y-tac-d commented:
良さそうなので一度マージします
